ios/podfile_helpers.rb

```diff
@@ -35,13 +35,16 @@ def run_versioned_postinstalls!(pod_name, target_installation_result)
     context: binding
 end
 
-def use_pods!(pattern, project_name = nil)
+def use_pods!(pattern, project_name = nil, excludePattern = "")
   base_directory = Pod::Config.instance.project_root
+  files_to_exclude = Dir.glob(excludePattern, base: base_directory)
```

Comment:
I'm wondering if running glob on empty string will be efficient — maybe use empty array in that case?
Another idea — what would you say to use an array of pod names to exclude instead of exclude pattern?

Reply:
> what would you say to use an array of pod names to exclude instead of exclude pattern?

Yeah, that's probably better. Might require more code if the list of pods grows long, but I don't think it will